lib/google_crawler/keywords.ex

```diff
@@ -72,8 +73,8 @@ defmodule GoogleCrawler.Keywords do
     |> group_by([keyword], keyword.id)
     |> select_merge([search_results: sr], %{
       result_count: count(sr.id),
-      ad_count: fragment("sum(?::int)", sr.is_ad),
-      top_ad_count: fragment("sum(?::int)", sr.is_top_ad)
+      ad_count: fragment("COALESCE(SUM(?::int), 0)", sr.is_ad),
```

Comment:
Could we somehow use this without fragments like `ad_count: type(sr.is_ad, :integer) |> sum() |> coalesce(0)`? 💭 

Just so I'm understanding it correctly, it's casting the search result bool to an integer, we're getting the sum, and if it's nil it's zero instead? 🤔

Reply:
@bterone yes, you understand correctly. If the value is nil, we need to make it 0 to make it filterable

And I think for this case using fragment is look simpler than use pipe 🤔